deepchem/feat/base_classes.py

```diff
@@ -262,7 +262,7 @@ def featurize(self, molecules, log_every_n=1000):
             "Failed to featurize datapoint %d. Appending empty array")
         features.append(np.array([]))
 
-    features = np.asarray(features)
```

Comment:
I think this line might be causing a lot of the Travis failures we see since for graph conv featurizations, we pass around `np.ndarray` objects with `dtype=object` (for `ConvMol`, `GraphData` etc).

Reply:
I reverted the change.  Let's see if I can find a more focused way of fixing the PyTorch error.